File: Modul 5/C/PRAK502-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h> 
#include <math.h> 
  
 int mutlak(int angka){ 
  
    if (angka < 0) { 
       angka *= -1; 
    } 
  
    return angka; 
  
 } 
  
 int hitung(int nilai1, int nilai2){ 
  
    int hitung; 
  
    hitung = nilai1 - nilai2; 
  
    if (hitung < 0) { 
       hitung *= -1; 
    } 
  
    return hitung; 
  
 } 
  
 int main() { 
  
  int a,b,c,d,Hasil,hasil1,hasil2; 
  
    printf(""); 
     scanf("%d %d %d %d",&a,&c,&b,&d); 
  
     Hasil = hitung(a,b) + hitung(c,d); 
     printf("%d", mutlak(Hasil)); 
  
  return 0; 
 }

File: Modul 5/Python/PRAK502-2310817310005-MuhammadNurwahyudiAdhitama.py
def mutlak(angka):
    if angka < 0:
        angka *= -1
    return angka

def hitung(nilai1, nilai2):
    hitung = nilai1 - nilai2
    if hitung < 0:
        hitung *= -1
    return hitung

a, c, b, d = map(int, input("").split())

hasil = hitung(a, b) + hitung(c, d)
hasil_mutlak = mutlak(hasil)

print(f"{hasil_mutlak}")
